setup.cfg

```diff
@@ -1,5 +1,18 @@
+[metadata]
+name = Tubes
+version = 0.2.0
+description = Flow control and backpressure for event-driven applications.
+license = MIT
+url = https://github.com/twisted/tubes/
+
+[options]
+packages = find:
+install_requires =
+    six
+    Twisted
+
 [bdist_wheel]
-universal=1
+universal = 1
 
 [flake8]
-ignore=E302,E303,E402
+ignore = E302,E303,E402
```

Comment:
Should this be `extend-ignore`?

Reply:
Ah yep this should be extend-ignore

Reply:
@graingert can you update the PR?

Reply:
I think we should fix this in a follow up. My plan was to make no semantic changes